packages/components/data-table-manager/src/data-table-manager.tsx

```diff
@@ -146,6 +162,33 @@ type TDataTableManagerProps<Row extends TRow = TRow> = {
   managerTheme?: 'light' | 'dark';
 };
 
+type TDataTableManagerContext = {
+  columns: TDataColumns;
+  updateColumns: (columns: TDataColumns) => void;
+};
+
+export const DataTableManagerContext = createContext<TDataTableManagerContext>({
```

Comment:
The context should live in a new file and also export a custom hook to access its information.

This will also help with having a better export from the package.